Add Crochet to the catalog

7h3kk1d opened the pull request:
Addressing https://github.com/malleable-systems/malleable.systems/issues/62

 I wanted to start with something but I have some issues with this entry as is.  I'm not sure the homepage is the best place to link to (the issue gives some alternative links but those also drop you into the middle of documentation. I grabbed a quote from the home page as well.

As an alternative I could extend the description to include some more interesting technical underpinnings and include some links to documentation there or it could be interesting to alter the catalog template to allow for multiple links.

Would love feedback from others on how to proceed. Thanks!

7h3kk1d committed:
Add Crochet to the catalog

jryans commented:
Hmm yeah, does seem like we need a bit more flexibility for extra links… As a quick fix for this entry, you could pack them in the description like a few others do (since it accepts Markdown). Feel free to investigate proper extra links in the template if you see a good way to structure it. 😄

7h3kk1d committed:
Add an additional resources section to template with values for Crochet

7h3kk1d commented:
> Hmm yeah, does seem like we need a bit more flexibility for extra links… As a quick fix for this entry, you could pack them in the description like a few others do (since it accepts Markdown). Feel free to investigate proper extra links in the template if you see a good way to structure it. smile

I added an `Additional Resources` . I'm a really bad designer so I wouldn't mind if someone wanted to do a better job. I also added another quote.

jryans commented on data/catalog.yaml:
```diff
@@ -770,3 +770,35 @@ works:
+    curators:
+      - 7h3kk1d
+    description:
+      "Crochet is a tool for creating and remixing interactive stories, safely"
```
```suggestion
      "Crochet is a tool for creating and remixing interactive stories, safely."
```

jryans commented on layouts/catalog/list.html:
```diff
@@ -86,6 +86,14 @@ <h1 id="works">
+              <span class="b mv0">Additional Resources</span>
+              <ul>
+              {{ range . }}
+                <li><a href="#{{ .url }}">{{ .label }}</a>{{ if .description }}: {{ .description }} {{ end }}</li>
```
We should leave off the `#` since these are external links.

```suggestion
                <li><a href="{{ .url }}">{{ .label }}</a>{{ if .description }}: {{ .description }} {{ end }}</li>
```

jryans commented on layouts/catalog/list.html:
```diff
@@ -86,6 +86,14 @@ <h1 id="works">
               {{ end }}
             </ul>
             {{ end }}
+            {{ with .additional_resources }}
```
Thanks for working on this! I am also not a designer by any stretch... Perhaps it's obvious from my total avoidance of colour... 😅 

Anyway, it's all quite subjective, but perhaps it's a bit better above the quotes, so that it's a bit closer to the "main" link. I'd also suggest making the resources header and the resources list a tad smaller, you can use the class `f6` for this. I think the `mv0` on the header can be removed, as it doesn't seem to do anything at the moment.

Here's a sample with those tweaks in place:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/279572/148246912-5affe31e-acc6-422b-a0cb-ce73c4ec705f.png">

Anyway, that's just my opinion, please let me know if you prefer your original styling. 🙂

7h3kk1d replied:
Yeah, I think that looks better. Let me know if you think anything else would be a good change

jryans reviewed (changes requested):
Seems like a good feature to have overall, just a few tweaks suggested. Thanks for working on this! 😄

7h3kk1d committed:
Add trailling period

Co-authored-by: J. Ryan Stinnett <jryans@gmail.com>

7h3kk1d committed:
Remove pound symbol

Co-authored-by: J. Ryan Stinnett <jryans@gmail.com>

7h3kk1d committed:
Move Additional Resources above quotes

jryans reviewed (approved):
Looks great, thanks for working on this! 😄
